examples/pos/arrays.effekt

```diff
@@ -18,12 +18,12 @@ def main() = {
     put(arr, 4, "foo");
     println(show(get(arr, 4)))
 
-    val arrWithVals = fill(4, "Hello World");
+    val arrWithVals: Array[String] = fill(4, "Hello World");
     println(arrWithVals)
 
-    val arrByF = build(5){ i => i };
+    val arrByF: Array[Int] = build(5){ i => i };
     println(arrByF);
 
-    var arrByList = fromList([1, 5, 17]);
+    var arrByList: Array[Int] = fromList([1, 5, 17]);
     println(arrByList)
```

Comment:
These annotations are necessary since we don't have qualified imports yet.
Thus the compiler cannot distinguish between:
```
fill[T](Int, T): Array[T]
```
and
```
fill[T](Int, T): List[T]
```